### PR TITLE
Add more details to encourage successful RFCs

### PR DESCRIPTION
**Pull request by benmccann**
We've gotten a good handful of RFCs over the past few days (woohoo!) and several people have commented both as authors and reviewers that they think we could improve things a bit. I think two main things have come up:
* people feel like they have to invest a lot into designing a solution, but really a lot of the value is in defining the problem well, gathering input from multiple parties, etc. The solution itself isn't always complicated, but aligning around it can be
* there's not a lot of focus on the content around the design. If someone just drops a design, it can be hard to tell what problems are being solved, how they arrived at this solution vs others, etc.

I tried to be a guinea pig for the new section here: https://github.com/benmccann/rfcs/blob/css/text/0000-sapper-css.md

### CODE REVIEW

File: 0000-template.md
Comment (lukeed) [inline 0000-template.md:84]: I think this is where we need the most prodding, since questions are what reviewers will put forth. AKA, current workarounds (if any) and how they're not enough/acceptable.

Comment (benmccann) [reply 0000-template.md:84]: > current workarounds (if any) and how they're not enough/acceptable

I thought of this as being something to go in the motivation section. I had "What does it look like to accomplish some set of goals today and how could that be improved?", but could explicitly call out workarounds if you think that'd help clarify

Comment (lukeed) [reply 0000-template.md:84]: Ah yea, sorry. Maybe being more direct with "What are the current workarounds, if any" is a better prompt/trigger-word?

Comment (benmccann) [reply 0000-template.md:86]: cool. I just added that

File: README.md
Comment (Conduitry) [inline README.md:44]: This should probably be updated, but I'm not sure to what.

File: 0000-template.md
Comment (dummdidumm) [inline 0000-template.md:50]: You said you wanted people to not focus on solutions too soon, but this sounds like the opposite to me ("this should get into specifics and corner-cases").

File: README.md
Comment (benmccann) [reply README.md]: good catch! I took a stab at updating this

File: 0000-template.md
Comment (benmccann) [reply 0000-template.md]: yeah, this was something I had trouble with. I tried to encourage a lot of detail before getting to this point (e.g. motivation and background). I think at some point the corner cases have to get answered before final approval though. I just tried to update this section a bit more. Take a look and let me know what you think

Comment (dummdidumm) [reply 0000-template.md]: Sounds better! Maybe a final sentence like "you don't need to specify this all up front as many of the implementation details may arise during discussion, but at least put some initial thought into it".

Comment (benmccann) [reply 0000-template.md]: I think it would be useful for people to include as much of this as they can, but I don't want it to be a barrier. I updated the wording in this section to make these sound a bit more like suggestions instead of being mandatory. I also updated the PR description to better describe what I was trying to accomplish here

**Comment by sw-yx** [comment]
defining "stages of acceptance" of an RFC could also be helpful to give an idea of forward motion to both the requester and interested parties

**Review by antony — APPROVED** [review]
This looks great. I think the final section is particularly useful - basically expect to get out what you are prepared to put in.

**Comment by antony** [comment]
> defining "stages of acceptance" of an RFC could also be helpful to give an idea of forward motion to both the requester and interested parties

Agreed - I wonder if it's always as straight-forward as this right now though, RFCs have many paths to becoming something.

**Comment by sw-yx** [comment]
i mean just off the cuff:

- stage 0: proposed
- stage 1: one maintainer agrees (can be same as proposer)
- stage 2: maintainer consensus achieved
- stage 3: working PR made
- stage 4: PR merged

**Comment by benmccann** [comment]
I wonder if we're that consistent with the ordering of stages. A lot of folks make a working PR first or in parallel to demonstrate that it's possible or what the code would look like. I've also had changes get in without anyone approving the RFC

**Comment by sw-yx** [comment]
just a suggestion, if premature its fine :)

**Review by pngwn — APPROVED** [review]

**Review by dummdidumm — APPROVED** [review]

**Pull request merged**
